src/app/hooks/useTxErrorMessage.ts

```diff
@@ -3,7 +3,7 @@ import { TxError } from '../../oasis-nexus/api'
 
 export const useTxErrorMessage = (error: TxError | undefined): string | undefined => {
   const { t } = useTranslation()
-  if (!error) return undefined
+  if (!error || (!error.module && !error.code)) return undefined
```

Comment:
Why check for a module? I think simplification should suffice:

```suggestion
  if (!error?.code) return undefined
```

Reply:
> Why check for a module? 

Because I don't know if we can assume that all possible modules (i.e. sources of errors) are sane enough to use the 0 error code for success.

There might be some deranged modules that break the convention. 

When hiding error messages, I want to be as narrow an specific as possible. We know that no module and error code 0 is not a problem, so we can hide this, but nothing else. 

If we see more useless error messages later, we can also hide them later...

Reply:
Maybe add a TODO, if this ever gets resolved upstream.

Reply:
> Maybe add a TODO, if this ever gets resolved upstream.

Added TODO note, referencing the upstream/backend issue.